src/statuspage.test.ts

```diff
@@ -0,0 +1,32 @@
+import DefaultOperations from './defaultOperations';
+import StatusPage from './statuspage';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/zipper-team/statuspage/pullRequest?prid=7096534)
```suggestion
import StatusPage from "./statuspage";
```